quinn/split_columns.py

```diff
@@ -16,7 +15,7 @@ def split_col(  # noqa: PLR0913
     delimiter: str,
     new_col_names: list[str],
     mode: str = "permissive",
-    default: Optional[str] = None,
+    default: str | None = None,
```

Comment:
does the pipe operator for optional type hints work on python versions before 3.10? The minimum python version supported is 3.7 in this repo so unit tests might fail